code/PyBake/logging.py

```diff
@@ -0,0 +1,116 @@
+'''
+Logging facility of our bakery
+'''
+
+import sys
+from enum import IntEnum, unique
+import textwrap
+
+@unique
+class LogVerbosity(IntEnum):
```

Comment:
Maybe we should use the ezEngine conventions for logging here? It would look like this:

~~~{.py}
class LogLevel(IntEnum)
    Error          = 1, ///< An error message.
    SeriousWarning = 2, ///< A serious warning message.
    Warning        = 3, ///< A warning message.
    Success        = 4, ///< A success message.
    Info           = 5, ///< An info message.
    Dev            = 6, ///< A development message.
    Debug          = 7, ///< A debug message.
~~~